tests/functional/simple_selection/seed.sql

```diff
@@ -0,0 +1,119 @@
+create table {schema}.summary_expected (
```

Comment:
Seems like this file is duplicated across places

Reply:
Yes. We're going to be looking for opportunities to store these commonly repeated files elsewhere. There's a 'tests/fixtures' directory. Maybe a 'tests/data' directory?

Reply:
Some of the seed.sql files might do different things, so we'll have to look at each one and see if it's a repeat.